web/packages/teleport/src/Discover/Database/DeployService/AutoDeploy/AutoDeploy.tsx

```diff
@@ -343,6 +346,11 @@ const Heading = ({
   region: string;
   wantAutoDiscover: boolean;
 }) => {
+  const theme = useTheme();
+  let img = theme.type === 'light' ? awsEcsLight : awsEcsDark;
+  if (theme.isCustomTheme && theme.name === 'bblp') {
+    img = awsEcsBblp;
+  }
```

Comment:
If we end up needing something like this in more places, it'd be good to create a helper function for this or a hook.